pysrc/bytewax/inputs.py

```diff
@@ -3,12 +3,47 @@
 Use these to wrap an existing iterator which yields items.
 
 """
+
 import datetime
 import heapq
+
+from .bytewax import AdvanceTo, Send
+
 from dataclasses import dataclass
 from typing import Any, Callable, Iterable, Tuple
 
 
+def yield_epochs(fn: Callable):
```

Comment:
I think the type signature on this would be `yield_epochs(iter: Iterable[Tuple[int, Any]]) -> Iterable[Any]`? Perhaps in the future we could add a base class for `AdvanceTo` and `Send` like `InputAction` which would help explain the final `Any`.